modules/apigee-x-core/variables.tf

```diff
@@ -36,9 +36,16 @@ variable "ax_region" {
   type        = string
 }
 
+variable "disable_vpc_peering" {
+  description = "Flag that specifies whether the VPC Peering through Private Google Access should be disabled between the consumer network and Apigee."
+  type        = bool
+  default     = false
+}
+
 variable "network" {
   description = "Network (self-link) to peer with the Apigee tennant project."
```

Comment:
Can you also mention that his isn't needed for the non-peering mode

Reply:
Done